### PR TITLE
pass etcd endpoints within " "

### PR DESCRIPTION
**Pull request by Madhu-1**
providing etcd endpoints inside [] fails with error 

Error: dial tcp: address [127.0.0.1:2379]: missing port in address

Signed-off-by: Madhu Rajanna <mrajanna@redhat.com>

### CODE REVIEW

**Comment by Madhu-1** [comment]
@aravindavk @prashanthpai @kshlm  please review

File: doc/development-guide.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ Run the following command to dump all keys and values in etcd to stdout. Replace
 different ip and port.
 
 ```sh
-ETCDCTL_API=3 etcdctl get --prefix=true "" --endpoints=[127.0.0.1:2379]
+ETCDCTL_API=3 etcdctl get --prefix=true "" --endpoints="127.0.0.1:2379"
```
Comment (prashanthpai) [inline doc/development-guide.md]: I'm not sure why this one didn't work for you. All 3 syntax seem to work for me:

```sh
[ppai@gd2-1 glusterd2]$ ETCDCTL_API=3 etcdctl get --prefix=true "gluster-ae83875a-d463-4da2-b770-ad7c5eadf4ab/peers" --endpoints=[127.0.0.1:2379]
gluster-ae83875a-d463-4da2-b770-ad7c5eadf4ab/peers/c9fa8750-4038-4cb0-9666-7069afadae3f
{"ID":"c9fa8750-4038-4cb0-9666-7069afadae3f","Name":"gd2-1","PeerAddresses":["127.0.0.1:24008"],"ClientAddresses":["127.0.0.1:24007"],"MetaData":null}


[ppai@gd2-1 glusterd2]$ ETCDCTL_API=3 etcdctl get --prefix=true "gluster-ae83875a-d463-4da2-b770-ad7c5eadf4ab/peers" --endpoints="127.0.0.1:2379"
gluster-ae83875a-d463-4da2-b770-ad7c5eadf4ab/peers/c9fa8750-4038-4cb0-9666-7069afadae3f
{"ID":"c9fa8750-4038-4cb0-9666-7069afadae3f","Name":"gd2-1","PeerAddresses":["127.0.0.1:24008"],"ClientAddresses":["127.0.0.1:24007"],"MetaData":null}


[ppai@gd2-1 glusterd2]$ ETCDCTL_API=3 etcdctl get --prefix=true "gluster-ae83875a-d463-4da2-b770-ad7c5eadf4ab/peers" --endpoints=127.0.0.1:2379
gluster-ae83875a-d463-4da2-b770-ad7c5eadf4ab/peers/c9fa8750-4038-4cb0-9666-7069afadae3f
{"ID":"c9fa8750-4038-4cb0-9666-7069afadae3f","Name":"gd2-1","PeerAddresses":["127.0.0.1:24008"],"ClientAddresses":["127.0.0.1:24007"],"MetaData":null}
```

Comment (Madhu-1) [reply doc/development-guide.md]: for me, it's not working


[root@etcd2 ~]# ETCDCTL_API=3 etcdctl get --prefix=true "" --endpoints=[192.168.122.252:2379]
Error: dial tcp: address [192.168.122.252:2379]: missing port in address
[root@etcd2 ~]# 
[root@etcd2 ~]# 
[root@etcd2 ~]# ETCDCTL_API=3 etcdctl get --prefix=true "" --endpoints="192.168.122.252:2379"
gluster-085680b4-7855-4375-ab70-af6322add8a0/alive/45035e69-cbac-4b60-8c65-cf4476940b81


[root@etcd2 ~]#etcdctl --version
etcdctl version: 3.3.4
API version: 2

Comment (prashanthpai) [reply doc/development-guide.md]: Okay, I'm on an older version:

```
$ etcdctl --version
etcdctl version: 3.2.4
API version: 2
```

**Comment by prashanthpai** [comment]
retest this please

**Pull request merged**